pkg/apis/etcd/v1beta2/restore_types.go

```diff
@@ -43,8 +43,24 @@ type RestoreSpec struct {
 	// using this spec, restore operator will prepare the seed that
 	// etcd operator will pick up later.
 	ClusterSpec ClusterSpec `json:"clusterSpec"`
+
+	// BackupStorageType is the type of the backup storage which is used as RestoreSource.
+	// TODO: implement me.
+	BackupStorageType string `json:"backupStorageType"`
```

Comment:
I think we use `StorageType` in `backupSpec`? should we also use `StorageType` here so that the naming scheme is consistent?

Reply:
In Restore Spec, it indicates the backup storage type.

Reply:
in backupSpec, the `StorageType` is also backup storage type. why differ in naming?

Reply:
Because this is in Restore Spec, not backup spec.

Reply:
sure.